thumbfast.lua

```diff
@@ -631,9 +652,9 @@ local function shutdown()
     end
 end
 
-mp.observe_property("display-hidpi-scale", "native", watch_changes)
-mp.observe_property("video-out-params", "native", watch_changes)
-mp.observe_property("vf", "native", watch_changes)
+mp.observe_property("display-hidpi-scale", "native", watch_changes_debounce)
+mp.observe_property("video-out-params", "native", watch_changes_debounce)
```

Comment:
Debounce for `video-out-params` causes the first thumbnail to fail in most cases when switching files with the cursor on the timeline.  
We get the new resolution too late. Other debounce calls seem fine.